src/main/resources/assets/spectrum/patchouli_books/guidebook/en_us/entries/hints/dragonbone.json

```diff
@@ -17,7 +17,7 @@
       "type": "spectrum:hint",
       "title": "Concise Instructions",
       "cost": "spectrum:doombloom_seed#12",
-      "text": "Dragonbone Blocks can be damaged using powerful explosions. You will need something with quite a bit more explosion power than TNT, for example Incandescent Amalgam. Though you feel that strong magical explosions might have a similar effect breaching their protective aura."
+      "text": "Dragonbone Blocks can be damaged using powerful explosions. You will need something with quite a bit more explosion power than TNT, for example Incandescent Amalgam. Though you feel that strong magical explosions might have a similar effect, breaching their protective aura."
```

Comment:
Note to Daf - Is there a reason this hint does not link to the page for Incandescent Amalgam?

Reply:
Hint pages currently reveal the raw text char by char, which would include patchouli's formatting codes, breaking them in the process. Small oof, but I've never come around improving on that. That's why none of the hint texts use any formatting.